babel.config.js

```diff
@@ -34,7 +35,14 @@ module.exports = api => {
       ],
       '@babel/preset-typescript'
     )
+    plugins.unshift(
```

Comment:
This plugin seems to want to go to the top. It's likely needed for all the react-native plugins. We could create an override for this too although the glob patterns may be more complicated.